pyiron_base/jobs/datamining.py

```diff
@@ -233,7 +234,7 @@ def _get_new_functions(self, file: FileHDFio) -> Tuple[List, List]:
             new_system_functions = []
         return new_user_functions, new_system_functions
 
-    def create_table(self, file, job_status_list, enforce_update=False):
+    def create_table(self, file, job_status_list, processes=1, enforce_update=False):
```

Comment:
I guess the number of processes could be more easily be specified via `job.server.cores`.

Reply:
I just had to remind myself - this is already done below.